Coding/server/lib/Session/Session.h

```diff
@@ -142,8 +107,10 @@ class Session : public httpsserver::WebsocketHandler
 
     /**
      * Checks which sessions need to be invalidated because of timeout
+     * 
+     * @param[in] parameter Generic input parameter for FreeRTOS task
      */
-    static void handleSessionTimeout();
+    static void handleSessionTimeout(void* parameter);
```

Comment:
parameter is not used

Reply:
Method signature may not deviate from what FreeRTOS expects for task creation

Reply:
State this with (void)parameter; in the function. That means not used and is accepted by most static code checkers.

Reply:
Sorry, I'm not entirely sure what you mean?

Reply:
Fixed with ee714a1